tests/Oryx.Tests.Common/DockerCli.cs

```diff
@@ -134,11 +136,17 @@ public DockerCli(TimeSpan waitTimeForExit, IEnumerable<EnvironmentVariable> glob
                     // are called for each line that is written to the output.
                     standardOutputHandler: (sender, args) =>
                     {
-                        outputBuilder.AppendLine(args.Data);
+                        lock (Lock1)
```

Comment:
as we discussed, let's put the locks in the same scope as the object it is protecting

Reply:
reverted lock